paramiko/auth_handler.py

```diff
@@ -20,8 +20,13 @@
 `.AuthHandler`
 """
 
+import threading
+import os
+from ctypes import (cdll, Structure, c_ulong, c_void_p, byref, c_char,
+                    c_int)
```

Comment:
minor style nit: see below, this is usually indented instead of wrapped:

```python
from ctypes import (
    cdll, Structure, c_ulong, c_void_p, byref, c_char, c_int
)
```